build.gradle

```diff
@@ -104,11 +109,16 @@ jar {
 
     manifest {
         attributes (
-                "Implementation-Version": version,
-                "Implementation-Title": project.description,
+            "Implementation-Version": version,
+            "Implementation-Title": project.description,
         )
     }
-    
+
```

Comment:
missing statement from the HDFS storage driver, the "provided" dependencies should be packaged into the driver's jar file